tests/conftest.py

```diff
@@ -1185,6 +1185,16 @@ def cleanup_providers_manager():
         ProvidersManager()._cleanup()
 
 
+@pytest.fixture
+def check_deprecations():
+    from airflow.exceptions import AirflowProviderDeprecationWarning, RemovedInAirflow3Warning
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("error", AirflowProviderDeprecationWarning)
+        warnings.simplefilter("error", RemovedInAirflow3Warning)
+        yield
```

Comment:
Let's have a look.
I've added this fixture which could be enabled on module level by `pytestmark = pytest.mark.usefixtures("check_deprecations")`

Reply:
Change concept from `pytestmark` on module scope to exclusion file on module ./ class / method / function scope